app/assets/javascripts/views/mobile_menu.js

```diff
@@ -0,0 +1,45 @@
+(function(App) {
+
+  'use strict';
+
+  var MobileMenu = Backbone.View.extend({
+
+    el: 'body',
+
+    events: {
+      'click .btn-mobile-menu' : 'onClickToggleMenu',
```

Comment:
We have a little convention that I like to use when we are using a class to identify an element from JS, we add a prefix js- to that class to mean that should only be used in JS and no in CSS, this is very useful to detach styles from functionality. You will add different classes for CSS and JS.